docs/ajaxApi.rst

```diff
@@ -1272,7 +1272,7 @@ Here's a curl command sample:
 
 .. code-block:: guess
 
-   curl -k --data "session.id=34ba08fd-5cfa-4b65-94c4-9117aee48dda&ajax=pauseFlow&execid=303" https://localhost:8443/executor
+   curl -k --get --data "session.id=34ba08fd-5cfa-4b65-94c4-9117aee48dda&ajax=pauseFlow&execid=303" https://localhost:8443/executor
```

Comment:
Actually for 'pauseFlow' and 'resumeFlow' endpoints what we need to change is the endpoint documentation not the example. These endpoints change state in the server so they should be POST (to be more precise PUT but Azkaban doesn't support it).